### ISSUE TITLE
Resuse binder session across snippets

### ISSUE THREAD

**Issue by mr-bjerre** [ines/course-starter-python#13]
Is it possible to use the same binder session for multiple snippets? For some exercises I want to continue on previous solutions.

**Comment by ines** [comment]
Yes, this is the `isolateCells` setting here, which is `true` by default (new session for each cell, i.e. code block):

https://github.com/ines/course-starter-python/blob/57d8b1ac80caddcdb921e6794b4f6666344cc48e/src/components/juniper.js#L23

**Comment by mr-bjerre** [comment]
Cool but it doesn't seem to work?

I just did `a = 2` in one session and then `a` was not recognized in another window.

I get `Launching Docker container on mybinder.org...` each time I run code in a new codeblock which takes ~20 sec. No chance to have it faster?

**Comment by hfboyce** [comment]
Sorry @ines, I also tried this changing `isolateCells: true` in  `src/components/juniper.js` and I am still not having follow up cells a retaining the variables in the previous cell's environment. 
Is there anywhere I should look to debug this? I think this feature could be really useful especially for importing modules and data.
